apps/astarte_data_updater_plant/test/astarte_data_updater_plant/data_updater_test.exs

```diff
@@ -47,22 +47,17 @@ defmodule Astarte.DataUpdaterPlant.DataUpdaterTest do
   alias CQEx.Result, as: DatabaseResult
   alias Astarte.RPC.Protocol.VMQ.Plugin, as: Protocol
 
-  alias Astarte.RPC.Protocol.VMQ.Plugin.{
-    Call,
-    Delete,
-    GenericOkReply,
-    Disconnect,
-    Reply
-  }
-
-  @vmq_plugin_destination Protocol.amqp_queue()
-  @encoded_generic_ok_reply %Reply{reply: {:generic_ok_reply, %GenericOkReply{}}}
-                            |> Reply.encode()
+  setup :verify_on_exit!
 
   setup_all do
     {:ok, _client} = Astarte.DataUpdaterPlant.DatabaseTestHelper.create_test_keyspace()
     {:ok, _pid} = AMQPTestHelper.start_link()
 
+    Mox.stub_with(
+      Astarte.DataUpdaterPlant.RPC.VMQPlugin.ClientMock,
+      Astarte.DataUpdaterPlant.RPC.VMQPlugin.Client
+    )
```

Comment:
Do we really need to stub the mock with the actual implementation here?

Reply:
No !!